### PR TITLE
[AggregateTargetInstaller] Unique resources passed to the script generator

### PR DESCRIPTION
**Pull request by dtorres**
As requested in #3315, an implementation on the `AggregateTargetInstaller` rather than `CopyResourcesScript`

I couldn't device a way to create a test this under AggregateTargetInstaller for this. Any help is appreciated.

### CODE REVIEW

**Comment by segiddins** [comment]
:+1: once we've added a test

**Comment by segiddins** [comment]
Let's also add a changelog entry. Fwiw, it might be easiest to test this if you extract the creation of the `resources_by_config` hash into its own method, then you can just stub the return value of `resources` on the file accessors.

**Comment by dtorres** [comment]
Great idea! I'll do that.

File: lib/cocoapods/installer/target_installer/aggregate_target_installer.rb [hunk 1]
```diff
@@ -102,16 +102,11 @@ def create_bridge_support_file
...
+      # Uniqued Resources grouped by config
       #
-      # @note   The bridge support file needs to be created before the prefix
-      #         header, otherwise it will not be added to the resources script.
+      # @return [Hash{ Symbol => Array<Pathname> }]
       #
-      # @return [void]
-      #
-      def create_copy_resources_script
-        path = target.copy_resources_script_path
```
Comment (dtorres) [inline lib/cocoapods/installer/target_installer/aggregate_target_installer.rb]: This removed line makes every other test to fail.
PR #3337 adds it back.
Sorry again.

**Pull request merged**